src/components/modals.tsx

```diff
@@ -1,12 +1,6 @@
 import React from 'react';
-import { SearchModal } from '@components/modals/search';
 import { AddNetworkModal } from '@components/modals/add-network';
 
 export const Modals: React.FC = () => {
-  return (
-    <>
-      <SearchModal />
-      <AddNetworkModal />
-    </>
-  );
+  return <AddNetworkModal />;
```

Comment:
Do you think we even need to keep this file? Maybe just relocate the one modal component or do we intend to add to this pattern in the future?

Reply:
Not necessary if you think we will add to it later. 👍